src/changelog/cli/_posts/2023-02-06-1-28-0.md

```diff
@@ -0,0 +1,15 @@
+---
+modified_at: 2023-02-06 16:00:00
+title: 'CLI - New version: 1.28.0'
+github: 'https://github.com/Scalingo/cli/releases'
+---
+
+### Installation
+
+[https://cli.scalingo.com](https://cli.scalingo.com)
+
+### Changelog
+
+* chore(deps): upgrade github.com/Scalingo/go-scalingo/v6.2.0
```

Comment:
Why a CLI release with such a limited changelog? Is it really useful to release a new version for the CLI? Moreover I expect the changelog to be more understandable by customers. Maybe this upgrade adds a useful feature for our customers?

Reply:
This dependency upgrade allows the CLI to have a better interpretation of the new implemented 429 too many requests errors. Maybe we can wait for the next feature to be merged for the CLI https://github.com/Scalingo/cli/pull/882  ?

Reply:
If that's the only change I would release only a patch version of the CLI and the changelog would be something like:

```
* chore(deps): better management of "429 Too Many Requests" errors (upgrade github.com/Scalingo/go-scalingo/v6.2.0)
```